src/main/java/com/redhat/jenkins/plugins/ci/threads/TriggerThreadProblemAction.java

```diff
@@ -0,0 +1,7 @@
+package com.redhat.jenkins.plugins.ci.threads;
```

Comment:
this only used in the '.sv' file that I asked you to delete...

Reply:
I removed this. It's part of another (future) change.